src/code42cli/cmds/detectionlists/__init__.py

```diff
@@ -1,4 +1,30 @@
+import click
+
 from code42cli.cmds.shared import get_user_id
+from code42cli.output_formats import OutputFormat
+from code42cli.output_formats import OutputFormatter
+
+
+def list_employees(employee_generator, output_format, additional_header_items=None):
+    additional_header_items = additional_header_items or {}
+    header = {"userName": "Username", "notes": "Notes", **additional_header_items}
+    employee_list = []
+    for employees in employee_generator:
+        for employee in employees["items"]:
+            if employee["notes"] and output_format == OutputFormat.TABLE:
+                employee["notes"] = (
+                    employee["notes"].replace("\n", ". ").replace("\t", ". ")
```

Comment:
I don't know if we should assume that every newline/tab doesn't already have a period in the note. 

If someone added a note with a few sentences with periods separated by newlines, we'd be duplicating the periods. 

And this translation should also happen for csv output, otherwise it'll write the newlines and mess up the csv format too. 

Maybe we should just translate any newlines to explicit `\n` strings so the intent stays the same but it keeps it all on a single line for printing?

Reply:
![Screen Shot 2020-12-15 at 21 59 57](https://user-images.githubusercontent.com/19540978/102277960-e77a7e00-3f20-11eb-9237-19d6be53507e.png)


Newlines don't matter in CSVs

Your other points are valid however

Reply:
I did first try replace newlines with explicit `\n` but the table outputter never liked it